test/imp/test-stepf.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+IGNORE=$($KPROVE --haskell-backend-command "$KORE_REPL -r --repl-script run-stepf-repl-script-spec.k.repl" -d . -m VERIFICATION sum-spec.k)
```

Comment:
Please change this to use the new make rule from https://github.com/kframework/kore/pull/1742.